### ISSUE TITLE
Dialog listener can't receive new message

### ISSUE THREAD

**Issue by nctp1209** [QuickBlox/quickblox-android-sdk#494]
I'm using SDK version 3.4.1 and i try to implement QBChat to provide chat function between 2 application.
The problem is when i sent 2 message in group dialog, dialog listener can't receive the first message until i sent the second message. And it received only the first message.
I have tried 2 way to listen to new message (QBIncomingMessageManager and add chat dialog listener) but it doesn't work (AutoJoinDialog is enabled)
I read the log from Smack and found that 2 message is sent to my device but somehow, the listener doesn't work.
Here is my code for QBIncomingMessagesManager:
```
public void initIncomingMessage(QBChatService chatService) {
        QBIncomingMessagesManager incomingMessagesManager = chatService.getIncomingMessagesManager();
        incomingMessagesManager.addDialogMessageListener(new QBChatDialogMessageListener() {
            @Override
            public void processMessage(String s, QBChatMessage qbChatMessage, Integer integer) {
                Log.e("ChatListener", "NewMessage: " + qbChatMessage.toString());
            }

            @Override
            public void processError(String s, QBChatException e, QBChatMessage qbChatMessage, Integer integer) {

            }
        });
    }
```
And listener for a specific dialog:
```
QBRestChatService.getChatDialogById(dialogId).performAsync(
                new QBEntityCallback<QBChatDialog>() {
                    @Override
                    public void onSuccess(final QBChatDialog dialog, Bundle params) {
                        dialog.addMessageListener(new QBChatDialogMessageListener() {
                            @Override
                            public void processMessage(String dialogId, QBChatMessage message, Integer senderId) {
                                Log.e(TAG, "New message: " + message);
                                Log.e(TAG, "Sender: " + senderId);
                            }

                            @Override
                            public void processError(String dialogId, QBChatException exception, QBChatMessage message, Integer senderId) {
                            }
                        });
                    }

                    @Override
                    public void onError(QBResponseException responseException) {

                    }
                });
```

**Comment by tatanka987** [comment]
hi @nctp1209, your problem looks like a bug which was fixed in version 3.5.1, try use latest version for check your issue

**Comment by nctp1209** [comment]
@tatanka987 thanks for your advice. I haven't tried it in version 3.5.1 yet but I found that when i add extra property to my message, it will cause this problem. Is this the bug which was fixed in version 3.5.1?

**Comment by tatanka987** [comment]
@nctp1209 if this property was empty or null yes, it was fixed
